BARBERSHOP. Сделал по заданию 5 главы 1 курса. Сделал базовую стилизацию. Исправил замечания.

Nomad48JMS opened the pull request:
Исправил замечания. Исправил ошибки.

petitechatte committed:
Поменял название ученика

Nomad48JMS committed:
Написал шапку Барбершопа

Написвл шапку, удалил абзац из <body>

Nomad48JMS committed:
Добавил класс лишке

Добавил класс лишке

Nomad48JMS committed:
markup document

Разметка главной страницы

Nomad48JMS committed:
Update

Разметил главную страницу барбершопа.
Сделал дополнительные страницы сайта.

Nomad48JMS committed:
Summary

Делал разметку

Nomad48JMS committed:
Update

Доделал разметку карточек товара

Nomad48JMS committed:
Update

Доделал разметку карточки товара

Nomad48JMS committed:
Update

Сделал разметку прайса

Nomad48JMS committed:
Update

Исправил ссылки

Nomad48JMS committed:
Validation

Прогнал код 4 страниц через валидатор устранил ошибки

Nomad48JMS committed:
Images

Вырезал три изображения с макета

Nomad48JMS committed:
Exported images and styles

Экспортировал изображения и стили

Nomad48JMS committed:
Exporting images and connecting content images.

Экспорт изображений и подключение контентных изображений.

Nomad48JMS committed:
Made the forms on the page and finished the layout of the main page.

Сделал формы на странице и закончил разметку главной страницы.

Nomad48JMS committed:
Update

Сделал страницу с картой. Сделал страницу с контактами под ссылку.

Nomad48JMS committed:
Update map.html

Начал делать страницу под карту.

Nomad48JMS committed:
Made the names of the images readable. Removed unnecessary.

Сделал читабельными названия изображений. Удалил лишнее. Удалил идиотские txt с экспортом.

Nomad48JMS committed:
Changed some pages. Added descriptions. And tweaked some things.

Изменил некоторые страницы. Добавил описаний. И подправил некоторые вещи.

Nomad48JMS committed:
Correctly exported the files that svg mistakenly did. Rewrote file paths for images.

Правильно экспортировал те файлы что ошибочно сделал svg . Переписал пути файлов для изображений.

Nomad48JMS committed:
I missed one file. Exported again.

Один файл пропустил. Заново экспортировал.

Nomad48JMS committed:
Fixed notes on layout from the previous pull request.

Исправил замечания по верстке из предыдущего пул реквеста.

Nomad48JMS committed:
Made the markup for three pages login.html, contacts.html and map.html.

Сделал разметку трех страниц login.html , contacts.html и map.html.

Nomad48JMS committed:
Aligned formatting.

Выровнял форматирование.

Nomad48JMS committed:
I completed the markup of the barbershop for chapter 4 of the courses. Made a product filter form. Tested it works.

Доделал разметку барбершопа по 4 главе курсов. Сделал форму фильтра товаров. Протестировал работает.

Nomad48JMS committed:
Made on assignment 4 chapters of 1 course.

Причесал код. Сделал по заданию 4 главы 1 курса.Перенес поясняющие комментарии вверх страниц. Так удобней.
Проверь пожалуйста. Прогнал на валидаторе все работает. Спасибо!:)

petitechatte commented on contacts.html:
```diff
@@ -0,0 +1,29 @@
+      </section>
+    </main>
+  </body>
+</html>
```
Очень здорово, что ты не поленился сделать дополнительную страницу. Но она не соответствует своему назначению! Нажимая "на обратную связь", пользователь ожидает увидеть форму отправки сообщения, а не информацию о фирме.

petitechatte replied:
Кстати, у меня тут автоматически высвечивается ошибка об отсутствии пустой строки в конце файла.

petitechatte commented on contacts.html:
```diff
@@ -0,0 +1,29 @@
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <title>Document</title>
```
Непонятное название страницы.

petitechatte commented on index.html:
```diff
@@ -1,33 +1,33 @@
-         <ul class="user-navigation">
-           <li class="login-link"><a href="index.html" aria-label="Вход в учетную запись.">Вход</a></li>
-         </ul>
+        <img class="logo-main" src="img/index-logo.png" width="371" height="155" alt="Логотип барбершопа Бородинский">
```
С пересохранением изображений ты немного перестарался. Логотип Барбершопа как раз сделан в svg. И на внутренних страницах тоже. Нужно исправить.
А вот всевозможные прямоугольники вообще не нужно было вырезать из макета. Но это ты потом поймешь, когда начнешь стилизацию.

petitechatte replied:
Кнопка входа ведет на текущую страницу! У тебя же есть отдельная страница для входа.

petitechatte commented on index.html:
```diff
@@ -1,33 +1,33 @@
+        <ul class="site-navigation">
+          <li><a href="info.html" aria-label="Ссылка на страницу с информацией.">Информация</a></li>
+          <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
+          <li ><a href="price.html" aria-label="Ссылка на страницу с ипрайс листом.">Прайс-лист</a></li>
```
Поправь текст в aria-label.

petitechatte commented on index.html:
```diff
@@ -61,7 +61,7 @@ <h2>Новости</h2>
+      <section class="gallery">
         <h2>Фотогалерея</h2>
         <figure class="gallery-content">
           <a href="#"><img src="img/Interior.jpg" width="286" height="164" alt="Интерьер" aria-label="Фотография интерьер барбершопа."></a>
```
Уже писала замечание о названиях файлов. По критериям качества в названиях файлов не должно быть прописных букв.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
       <section class="appoimtment">
```
Опечатка в названии класса.

petitechatte replied:
Почему ссылка для обратной связи ведет на страницу контактов? Вряд ли пользователь будет звонить в Барбершоп, если, допустим, захочет сообщить, что на сайте не работает карта.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
-           <input type="text" name="phone" id="form-phone" value="" placeholder="+7 123 456-78-90" aria-label="Введите ваш телефон.">
-           <input type="submit" name="form-button" value="Отправить" aria-label="Кнопка отправки формы.">
-          </fieldset>
+        <form action="hhtp://echo.htmlacademy.ru" method="post" class="appoimtment-form" aria-label="Форма записи на прием к мастеру.">
```
Опечатка в адресе. Опечатка в названии класса.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
-          </fieldset>
+        <form action="hhtp://echo.htmlacademy.ru" method="post" class="appoimtment-form" aria-label="Форма записи на прием к мастеру.">
+          <label for="form-date">Дата:</label>
+          <input type="text" name="date" id="form-date" value="" placeholder="08.02.2017" aria-label="Введите дату посещения.">
```
После подписей к полям на макете нигде не стоят двоеточия!

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+        <ul>
+          <li><a class="social-button" href="https://vk.com/" aria-label="Барбершоп бородинский Вконтакте">Вконтакте</a></li>
+          <li><a class="social-button" href="https://www.facebook.com/" aria-label="Барбершоп бородинский в Facebook">Фэйсбук</a></li>
+          <li><a class="social-button" href="https://www.instagram.com/" aria-label="Барбершоп бородинский в Instagram">Инстаграм</a></li>
```
Это не ошибка, но в конце адреса можно не ставить "/".

petitechatte replied:
Еще потом тебе понадобятся индивидуальные классы для каждой кнопки, например, social-button-vk.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+      <p class="footer-copyright">
+      <b>Разработано</b>
+      <a class="button" href="https://htmlacademy.ru" aria-label="Ссылка на сайт Академии">HTML Academy</a>
+    </footer>
```
Футер лучше разбить на 3 дива (у тебя сейчас выделен только средний).

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+      </div>
+      <p class="footer-copyright">
+      <b>Разработано</b>
+      <a class="button" href="https://htmlacademy.ru" aria-label="Ссылка на сайт Академии">HTML Academy</a>
```
Тэг p не закрыт! Кстати, ты это не заметил из-за отсутствия отступов у вложенных тэгов.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+    <section class="modal modal-login">
+      <h2>Личный кабинет</h2>
+      <p>Введите свой логин и пароль</p>
+      <form action="hhtp://echo.htmlacademy.ru" method="post" class="modal-login-form" aria-label="Форма входа в учетную запись.">
```
Опять ошибка в адресе, из-за которой форма не отправляется!

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+      <p>Введите свой логин и пароль</p>
+      <form action="hhtp://echo.htmlacademy.ru" method="post" class="modal-login-form" aria-label="Форма входа в учетную запись.">
+        <input type="text" name="username" placeholder="ЛОГИН" value="" aria-label="Введите ваш логин.">
+        <input type="password" name="password" placeholder="ПАРОЛЬ" aria-label="Введите ваш пароль.">
```
Полям желательно прописать лейблы с классом visually-hidden. Хотя у тебя есть aria-label, возможно этого достаточно, я точно не знаю.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+      <form action="hhtp://echo.htmlacademy.ru" method="post" class="modal-login-form" aria-label="Форма входа в учетную запись.">
+        <input type="text" name="username" placeholder="ЛОГИН" value="" aria-label="Введите ваш логин.">
+        <input type="password" name="password" placeholder="ПАРОЛЬ" aria-label="Введите ваш пароль.">
+        <input type="checkbox" name="login-radio" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке.">
```
Очень странное имя для поля. 
Значение атрибута name уходит на сервер, выглядит это примерно так:
login-radio: yes
Абсолютно никакой смысловой нагрузки. Тем более, у тебя тут даже нет радиокнопки. Значение id тоже стоит изменить.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+        <input type="password" name="password" placeholder="ПАРОЛЬ" aria-label="Введите ваш пароль.">
+        <input type="checkbox" name="login-radio" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке.">
+        <label for="modal-login-radio">Запомните меня!</label>
+        <a href="login.html" aria-label="Ссылка на страницу восстановления пароля.">Я забыл пароль!</a>
```
Я заполняла форму входа, забыла пароль и решила его восстановить, перейдя по сссылке... и опять попала на форму входа! Какой-то замкнутый круг...
Если у тебя нет страницы для формы восстановления пароля, просто поставь заглушку.
Кстати, было бы логично сделать две отдельные страницы - для входа и для регистрации отдельно, чтобы не путать пользователя. Но на макете регистрация почему-то вообще не продумана...

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
-          <input type="submit" value="ВОЙТИ" aria-label="Кнопка входа в учетную запись.">
+    <section class="modal modal-login">
+      <h2>Личный кабинет</h2>
+      <p>Введите свой логин и пароль</p>
```
Текст не соответствует макету!

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
-      </section>
+    <section class="modal modal-map">
+      <h2 class="visualy-hidden">Как до нас добраться</h2>
+      <img src="img/map-popup.png" width="782" height="577" alt="Карта, на которой показано, как к нам проехать.">
```
В атрибуте alt абсолютно бесполезная информация для незрячих. Да и зрячих этот текст не слишком порадует, если карта не загрузится. Лучше прямо сюда прописать адрес фирмы.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+    <section class="modal modal-map">
+      <h2 class="visualy-hidden">Как до нас добраться</h2>
+      <img src="img/map-popup.png" width="782" height="577" alt="Карта, на которой показано, как к нам проехать.">
+      <button class="modal-close" type="button" aria-label="Кнопка закрытия окна карты"><img src="img/cross.png" width="19" height="19" alt="Кпопка крест для закрытия модального окна"></button>
```
Здесь те же проблемы с кнопкой: неправильный формат картинки, желателен текст, хотя, возможно, aria-label спасает ситуацию.

petitechatte commented on contacts.html:
```diff
@@ -0,0 +1,29 @@
+
+  <body>
+    <main class="main-container-contacts">
+      <h1>Контакты нашей фирмы.</h1>
```
После заголовков точка не ставится.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+        <a href="login.html" aria-label="Ссылка на страницу восстановления пароля.">Я забыл пароль!</a>
+        <button class="button form-apointment-button" type="submit" aria-label="Кнопка входа в учетную запись.">Войти</button>
+      </form>
+      <button type="button" class="modal-close" aria-label="Кнопка закрытия окна входа на сайт."><img src="img/cross.svg" width="19" height="19" alt="Кпопка крест для закрытия модального окна"></button>
```
Кнопка не отображается, так как ты ссылаешься на несуществующий файл. Кстати, этот крестик в макете как раз сделан векторной графикой. Пересохрани его заново.

В значении атрибута alt опечатка!

Я не помню точно, как правильно верстаются кнопки, чтобы они были максимально доступными, но по-моему, нужно добавить кнопке текст "Закрыть", а крестик сделать фоном.

petitechatte commented on item.html:
```diff
@@ -1,15 +1,19 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
     <title>Барбершоп Бородинский товары</title>
```
Почему "товары"? Это страница конкретного товара, его и нужно прописать в title. В реальном интернет-магазине title будет меняться автоматически, но тем не менее, сейчас это выглядит как косяк.

petitechatte commented on item.html:
```diff
@@ -1,15 +1,19 @@
       <nav class="main-navigation" aria-label="Навигация сайта.">
         <a class="main-header-logo" href="index.html" aria-label="Логотип ссылка на главную страницу.">
-        <img src="img/logo.svg" width="111" height="24" alt="Логотип барбершопа Бородинский">
+        <img src="img/logo.png" width="111" height="24" alt="Логотип барбершопа Бородинский">
```
Этот логотип тоже векторный! Пересохрани.
И кстати, не забывай про отступы вложенных тегов!

petitechatte commented on item.html:
```diff
@@ -1,15 +1,19 @@
   <body>
     <header class="main-header">
       <nav class="main-navigation" aria-label="Навигация сайта.">
         <a class="main-header-logo" href="index.html" aria-label="Логотип ссылка на главную страницу.">
```
С точки зрения русского языка, aria-label написан безграмотно.

petitechatte commented on item.html:
```diff
@@ -1,15 +1,19 @@
+        <img src="img/logo.png" width="111" height="24" alt="Логотип барбершопа Бородинский">
         </a>
         <ul class="site-navigation">
           <li><a href="info.html" aria-label="Ссылка на страницу с информацией.">Информация</a></li>
```
У тебя нет страницы с информацией, лучше оставить заглушку, чтобы не переходить на пустую страницу. 

В принципе, не очень понятно, зачем здесь aria-label. Программа-читалка и так понимает, что это ссылка, и озвучит текст внутри нее. По моему, этого должно быть достаточно.

petitechatte commented on item.html:
```diff
@@ -23,8 +27,8 @@
         </ul>
```
То же самое касается остальных ссылок. Незрячие услышат ту же информацию, что и увидят зрячие. На мой взгляд, aria-label тут не нужен, так как сам текст внутри ссылки - это уже лейбл. Если текста нет, тогда да, нужны дополнительные меры обеспечения доступности.

Ссылку на новости тоже лучше поставить заглушкой.

В aria-label для прайс-листа опечатка, которая у тебя будет на всех страницах.

Ссылка на магазин ведет непонятно куда. Учитывая ограниченное количество страниц, можно поставить ссылку на shop.html, просто для удобства просмотра твоей работы. Тем более, что есть такой критерий качества.

Почему кнопка входа ведет на главную?!

petitechatte replied:
Почему кнопка входа все еще ведет на главную?!

petitechatte commented on contacts.html:
```diff
@@ -0,0 +1,29 @@
+      </section>
+
+      <section class="working-hours">
+        <h2>Время работы барбершопа.</h2>
```
После заголовков точка не ставится.

petitechatte commented on item.html:
```diff
@@ -35,7 +39,6 @@ <h1 class="page-title">Набор для путешествий «Baxter of Cali
 
```
Ссылка "Магазин" должна вести в магазин, а не на текущую страницу. Такой страницы нет, поэтому нужно ставить заглушку.

В "крошках" пропущена ссылка! Пользователь не сможет вернуться в предыдущий раздел.

petitechatte replied:
По ссылкам. Логичнее было бы поставить ссылку на "Средства для ухода". Но сами макеты сделаны не очень логично, поэтому можешь оставить так.

petitechatte commented on item.html:
```diff
@@ -55,7 +58,6 @@ <h2 class="visualy-hidden">Описание товара</h2>
           <b>2900 р</b>
           <a class="button" href="#">Купить</a>
```
product-prise - неподходящее название класса. Товар-приз?!

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+      <p class="footer-contact">
+        Барбершоп «Бородинский»<br>
+        Адрес: г. Санкт-Петербург, Б. Конюшенная, д. 19/8<br>
+      </p>
```
Не соответствует макету.
Где ссылка и телефон?

petitechatte commented on item.html:
```diff
@@ -64,11 +66,14 @@ <h3>В набор входят:</h3>
+        <p class="footer-contact">
+          Барбершоп «Бородинский»<br>
+          Адрес: г. Санкт-Петербург, Б. Конюшенная, д. 19/8<br>
+        </p>
```
Замечания по футеру описаны для главной страницы. Нужно будет поправить везде.

petitechatte commented on item.html:
```diff
@@ -85,23 +90,22 @@ <h3>В набор входят:</h3>
-        <button class="modal-close" type="button" aria-label="Кнопка закрытия окна карты"><img src="img/cross-map-popup.svg" width="19px" height="19px"></button>
+        <img src="img/map-popup.png" width="782" height="577" alt="Карта, на которой показано, как к нам проехать.">
+        <button class="modal-close" type="button" aria-label="Кнопка закрытия модального окна."><img src="img/cross.png" width="19" height="19" alt="Кпопка крест для закрытия модального окна"></button>
       </section>
```
Замечания по модальным окнам описаны для главной страницы.

petitechatte commented on item.html:
```diff
@@ -85,23 +90,22 @@ <h3>В набор входят:</h3>
       </section>
   </body>
-</html>
+</html>
```
GitHub отмечает как ошибку отсутствие пустой строки в конце файла.

petitechatte commented on index.html:
```diff
@@ -85,57 +85,56 @@ <h2>Контактрая информация</h2>
+    </section>
   </body>
-</html>
+</html>
```
Отсутствие пустой строки в конце файла.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+      <h1>Форма регистрации учётной записи и форма входа</h1>
+      <section class="account-login-form">
+        <h2>Форма регистрации</h2>
+        <form action="hhtp://echo.htmlacademy.ru" method="post" class="form-registration">
```
Неправильный адрес, форма не отправляется.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+      <section class="account-registration-form">
+        <h2>Форма входа в учетную запись</h2>
+        <p>Введите свой логин и пароль</p>
+          <form action="hhtp://echo.htmlacademy.ru" method="post" class="modal-login-form" aria-label="Форма входа в учетную запись.">
```
Неправильный адрес, форма не отправляется.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+        <h2>Форма регистрации</h2>
+        <form action="hhtp://echo.htmlacademy.ru" method="post" class="form-registration">
+          <label for="registration-email">Введите ваш E-mail</label>
+          <input type="text" id="registration-email" placeholder="Nomad48JMS@gmail.com" aria-label="Введите свой email в форму">
```
Нельзя писать свой реальный e-mail!!! 
Это же образец, достаточно в таком виде: email@gmail.com или vasyapupkin@gmail.com

Aria-label тут избыточен, по-моему. Есть же настоящий label, и в нем уже написано, что делать.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+  <body>
+    <main class="main-container-login">
+      <h1>Форма регистрации учётной записи и форма входа</h1>
+      <section class="account-login-form">
```
Класс не соответствует содержимому.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+        </form>
+      </section>
+
+      <section class="account-registration-form">
```
Класс не соответствует содержимому.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+            <label for="modal-login-radio">Запомните меня!</label>
+            <a href="login.html" aria-label="Ссылка на страницу восстановления пароля.">Я забыл пароль!</a>
+            <button class="button form-apointment-button" type="submit" aria-label="Кнопка входа в учетную запись.">Войти</button>
+          </form>
```
Замечания по форме уже описаны для главной страницы.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+            <button class="button form-apointment-button" type="submit" aria-label="Кнопка входа в учетную запись.">Войти</button>
+          </form>
+      </section>
+      <a href="index.html" aria-label="Ссылка переход на главную">На главную.</a>
```
Aria-label безграмотный, с точки зрения русского языка. Точка в ссылке нежелательна.

petitechatte commented on login.html:
```diff
@@ -0,0 +1,42 @@
+      <a href="index.html" aria-label="Ссылка переход на главную">На главную.</a>
+    </main>
+  </body>
+</html>
```
Отстутствие пустой строки в конце файла.

petitechatte commented on map.html:
```diff
@@ -1,12 +1,22 @@
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <title>Карта проезда Барбершоп Бородинский.</title>
```
Точка в заголовке не ставится. "Бородинский" лучше взять в кавычки, как это пишется в других местах сайта. И в целом, название смотрится странно, лучше переделать.

petitechatte commented on map.html:
```diff
@@ -1,12 +1,22 @@
+
+  <body>
+    <main class="main-container-map">
+      <h1 class="visually-hidden">Как к нам проехать</h1>
```
Почему visually-hidden? Это уже не модальное окно. Заголовок здесь вполне уместен.

petitechatte commented on map.html:
```diff
@@ -1,12 +1,22 @@
+    <main class="main-container-map">
+      <h1 class="visually-hidden">Как к нам проехать</h1>
+      <section class="barbershop-address">
+        <h2>Адресс Барбершопа Бородинский</h2>
```
Опечатка в тексте. Нужны кавычки. Не думаю, что барбершоп нужно писать с большой буквы.

petitechatte commented on map.html:
```diff
@@ -1,12 +1,22 @@
+      <h1 class="visually-hidden">Как к нам проехать</h1>
+      <section class="barbershop-address">
+        <h2>Адресс Барбершопа Бородинский</h2>
+        <img src="img/map-borodinsky.png" width="766" height="560" alt="Адресс г. Санкт-Петербург, Б. Конюшенная, д. 19/8" aria-label="Карта, где показано, как к нам проехать.">
```
Адрес пишется с одной "с"!
После слова адрес нужно поставить двоеточие. А можно вообще его убрать, сверху же есть заголовок, который поясняет, что это.

Не вижу смысла в использовании alt и aria-label для одного и того же тега. Alt вполне справляется со своими функциями.

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
     <title>Барбершоп Бородинский цены</title>
```
Стоит поработать над заголовками. Выглядят не очень, а ведь они отображаются в поисковых запросах!

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
       <nav class="main-navigation">
         <a class="main-header-logo" href="index.html">
-        <img src="img/logo.svg" width="111" height="24" alt="Логотип барбершопа Бородинский">
+        <img src="img/logo.png" width="111" height="24" alt="Логотип барбершопа Бородинский">
```
Неправильный формат для векторной графики. Нет отступа для вложенного тега.

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
-           <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
-          <li class="site-navigation-current"aria-label="Ссылка на текущую страницу с ипрайс листом." ><a href="#">Прайс-лист</a></li>
+          <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
+          <li class="site-navigation-current" ><a href="price.html" aria-label="Ссылка на текущую страницу с ипрайс листом">Прайс-лист</a></li>
```
Ссылки на текущую страницу делать не нужно! Какой в ней смысл?

Остальные замечания по навигации я уже писала для главной.

petitechatte replied:
Здесь нужно оставить ссылку, т.к. она появляется на других страницах, но убрать href, чтобы ссылка не была активной.

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
+          <li><a href="contacts.html" aria-label="Ссылка на страницу с контактами.">Контакты</a></li>
+        </ul>
+        <ul class="user-navigation">
+          <li class="login-link"><a href="index.html" aria-label="Вход в учетную запись.">Вход</a></li>
```
Опять ссылка ведет не туда.

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
       <ul class="breadcrumbs">
         <li><a href="index.html">Главная</a></li>
-        <li class="breadcrumbs-current"><a href="#">Прайс-лист</a></li>
+        <li class="breadcrumbs-current"><a href="price.html">Прайс-лист</a></li>
```
Внимательно читай ТЗ:
3.3. В блоке с хлебными крошками активная страница (последний пункт) не является ссылкой и не реагирует на наведение.

petitechatte commented on price.html:
```diff
@@ -1,33 +1,38 @@
+        <li class="breadcrumbs-current"><a href="price.html">Прайс-лист</a></li>
       </ul>
       <b class="product-slogan">Истинно мужская класика</b>
       <section class="servises">
```
Опечатка в названии класса.

petitechatte replied:
Ниже тоже опечатки в слове brand. Также не очень понятно, при чем тут resource...

Вообще, это пример страницы, которую будет заполнять контент-менеджер. Здесь нужно использовать классы по минимуму. Чем отличается этот список от любого другого списка на сайте? Например, от списка компонентов в наборе?

petitechatte commented on price.html:
```diff
@@ -66,35 +71,35 @@ <h2>Несколько слов о нас:</h2>
           чувствовать себя комфортно и свободно!</p>
       </section>
     </main>
+
```
"Цены на услуги <br> наших мастеров:" - здесь не очень понятно по макету, хочет ли дизайнер именно такой перенос или ему просто важна ширина заголовка (например, по ширине столбца таблицы).
Учитывая то, что это контентная страница, заполняемая вручную контент-менеджером, предпочтительнее второй вариант, без тега br.

Пустой класс в таблице!

Рубли, видимо, нужно вставлять спецсимволом-мнемоникой.

Классы для абзацев тоже не нужны. Лучше повесить особый класс на main или body.

Тег br в тексте - очень, очень грубая ошибка!

Когда будешь писать стили, эту страницу пока не трогай. Она особенная.

petitechatte commented on price.html:
```diff
@@ -66,35 +71,35 @@ <h2>Несколько слов о нас:</h2>
-          <b>Разработано</b>
-          <a class="button" href="https://htmlacademy.ru" aria-label="Ссылка на сайт Академии">HTML Academy</a>
-      </footer>
+      <p class="footer-contact"></p>
```
Несоответствие макету: нет контента!

petitechatte commented on price.html:
```diff
@@ -66,35 +71,35 @@ <h2>Несколько слов о нас:</h2>
+        <button class="button form-apointment-button" type="submit" aria-label="Кнопка входа в учетную запись.">Войти</button>
+      </form>
+      <button type="button" class="modal-close" aria-label="Кнопка закрытия окна входа на сайт."><img src="img/cross.svg" width="19" height="19" alt="Кпопка крест для закрытия модального окна"></button>
+    </section>
```
Замечания по модальным окнам описаны для главной.

petitechatte replied:
Кстати, а почему нет модального окна с картой?
Я так понимаю, что карта открывается ссылкой из подвала, а значит, должна быть на каждой странице.

petitechatte commented on price.html:
```diff
@@ -66,35 +71,35 @@ <h2>Несколько слов о нас:</h2>
+    </section>
   </body>
-</html>
+</html>
```
Отсутствие пустой строки в конце файла.

petitechatte commented on shop.html:
```diff
@@ -1,25 +1,29 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
     <title>Барбершоп Бородинский магазин</title>
```
Но ведь это страница средств для ухода?

petitechatte commented on shop.html:
```diff
@@ -1,25 +1,29 @@
-           <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
+          <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
           <li ><a href="price.html" aria-label="Ссылка на страницу с ипрайс листом.">Прайс-лист</a></li>
           <li class="site-navigation-current"><a href="catalog.html" aria-label="Ссылка на текущую страницу магазина.">Магазин</a></li>
```
Не должно быть ссылки на текущую страницу. В принципе, данная странница не совсем магазин, а его раздел. В этом случае в ссылке должна стоять заглушка (у тебя нет страницы магазина). Либо убрать href вообще.

petitechatte replied:
Остальные замечания по шапке, как и для главной.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
 
```
Ссылка в магазин почему-то ведет на страницу конкретного товара!

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
       <section class="filters">
         <h2 class="visualy-hidden">Фильтр товаров</h2>
-        <!-- здесь будет форма -->
+        <form action="hhtp://echo.htmlacademy.ru" method="get">
```
Неправильный адрес, форма не работает!

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
+        <form action="hhtp://echo.htmlacademy.ru" method="get">
+          <fieldset>
+            <legend>Производители:</legend>
+            <input type="checkbox" name="baxter-of-california" value="baxter-of-california" id="baxter-of-california" checked aria-label="Добавить или убрать Baxter of California из фильтра">
```
Aria-label здесь избыточен и он очень сложный. У тебя уже есть привязанный к полю label.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
+            <label for="baxter-of-california">Baxter of California</label>
+            <input type="checkbox" name="mr-natty" value="" id="mr-natty" aria-label="Добавить или убрать Mr Natty из фильтра">
+            <label for="mr-natty">Mr Natty</label>
+            <input type="checkbox" name="suavecito" value="" id="suavecito" checked aria-label="Добавить или убрать Suavecito из фильтра">
```
Просто рекомендация: атрибут checked лучше ставить самым последним, тогда он заметнее в коде.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
-            <a href="catalog-item.html">
-             <span class="catalog-category">Набор для путешествий</span>
-             <span class="catalog-item-title">«Baxter of California»</span>
+        <li class="catalog-item">
```
Нет отступа вложенного тега.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
+        <li class="catalog-item">
+          <h3>
+            <a href="#">
+            <span class="catalog-category">Набор для путешествий</span>
```
Нет отступов вложенных тегов.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
-             <span class="catalog-item-title">«Baxter of California»</span>
+        <li class="catalog-item">
+          <h3>
+            <a href="#">
```
У тебя есть страница для этого товара! Сделай ссылку на нее!

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
-           </p>
+          </h3>
+          <p class="catalog-item-image">
+            <a href="#">
```
Здесь тоже нужно поставить конкретную ссылку на сверстанную страницу.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
+          <p class="catalog-item-price">
+            <b>2900 р</b>
+            <a class="button" href="#">Купить</a>
+          </p>
```
Нет отступа для вложенных тегов.

petitechatte commented on item.html:
```diff
@@ -55,7 +58,6 @@ <h2 class="visualy-hidden">Описание товара</h2>
           <b>2900 р</b>
```
В макете используется спецсимвол для рубля.

petitechatte commented on shop.html:
```diff
@@ -35,40 +39,64 @@ <h1 class="page-title">Средства для ухода</h1>
+            </a>
+          </p>
+          <p class="catalog-item-price">
+            <b>2900 р</b>
```
Единственная карточка, где ты почему-то не используешь спецсимвол для рубля.

petitechatte commented on shop.html:
```diff
@@ -146,14 +174,15 @@ <h3>
     </main>
+
     <footer class="main-footer">
         <p class="footer-contact"></p>
```
Пустой раздел, нет контента!

petitechatte commented on shop.html:
```diff
@@ -169,21 +198,26 @@ <h3>
+        <h2 class="visualy-hidden">Как до нас добраться</h2>
+        <img src="img/map-popup.png" width="782" height="577" alt="Карта, на которой показано, как к нам проехать.">
+        <button class="modal-close" type="button" aria-label="Кнопка закрытия окна карты"><img src="img/cross.png" width="19" height="19" alt="Кпопка крест для закрытия модального окна"></button>
       </section>
```
Замечания по модальным окнам описаны для главной страницы.

petitechatte commented on shop.html:
```diff
@@ -169,21 +198,26 @@ <h3>
 
   </body>
-</html>
+</html>
```
Отсутствие пустой строки в конце файла.

petitechatte replied:
Важно! Тебе нужно создать в проекте файл .gitignore (да, именно так, без названия, только расширение).
В этом файле для твоего проекта достаточно добавить одну строку: /layout
Тогда эта папка будет лежать в твоем проекте, но git не будет ее обрабатывать.

petitechatte reviewed (changes requested):
Нужно еще раз внимательно изучить тему по изображениям и пересохранить картинки.

Nomad48JMS committed:
Добавил .gitignore для папки с макетами. Подправил некоторые ошибки.

Добавил .gitignore для папки с макетами. Подправил некоторые ошибки.

Nomad48JMS committed:
Удалил ненужные страницы.

Удалил ненужные страницы.

Nomad48JMS committed:
Удалил ненужные файлы. Добавил отступ снизу. Начал исправлять ошибки и переделывать классы.

Удалил ненужные файлы. Добавил отступ снизу. Начал исправлять ошибки и переделывать классы.

Nomad48JMS committed:
Delete advantages-marker.svg

нарисую потом

Nomad48JMS committed:
Удалил пару ненужных файлов. Переработал 3/4 страниц проекта с учетом замечаний и по БЭМ. Проверил на валидаторе. Сверил разметку в соответствии с макетом.

Удалил пару ненужных файлов. Переработал 3/4 страниц проекта с учетом замечаний и по БЭМ. Проверил на валидаторе. Сверил разметку в соответствии с макетом.

Nomad48JMS committed:
Merge branch 'master' of https://github.com/Nomad48JMS/sanya-learning

Nomad48JMS committed:
Переделал весь проект по БЭМ. Исправил замечания и доработки. Проверил на валидацию и ошибки.

Переделал весь проект по БЭМ. Исправил замечания и доработки. Проверил на валидацию и ошибки.

Nomad48JMS committed:
Пару отступов убрал

Пару отступов убрал

Nomad48JMS committed:
Changed markup, classes. Changed some of the images. Made a piece of basic styling.

Изменил разметку, классы. Изменил часть изображений. Сделал кусок базовой стилизации.

Nomad48JMS committed:
Made the basic styling of the index page. I changed three images. Fixed classes. Checked the validation.

Сделал базовую стилизацию индексной страницы. Поменял три изображения. Исправил классы. Проверил валидацию.

Nomad48JMS committed:
Made on assignment 5 chapter 1 course

Сделал по заданию 5 главы 1 курса

Nomad48JMS committed:
Corrected the image. Removed effects.

Исправил изображение. Убрал эффекты.

Nomad48JMS committed:
Update style.css

Пробел в конце

Nomad48JMS committed:
Update settings.json

Пробел в конце

Nomad48JMS committed:
Пробелы

Пробелы

Nomad48JMS committed:
ver

petitechatte commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+#������������ ����� �� ����� layout
```
что это?

petitechatte commented on README.md:
```diff
@@ -1,24 +1,70 @@
+Название сайта: Барбершоп «Бородинский»
 
-#### 1. Зарегистрируйтесь на Гитхабе
+Домен: https://zloydedd.github.io/Barbershop/
```
это как?

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+    <link href="css/style.css" rel="stylesheet">
   </head>
+
   <body>
```
Главная страница отличается от остальных фоном, поэтому ей можно дать свой класс.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+
   <body>
+    <header class="main-header">
+      <nav class= "main-header__main-navigation container-centerer">
```
Не уверена, что есть такое слово "centerer". Обычно используют просто "container" или "wrapper". Список типичных классов можно посмотреть [здесь](https://github.com/yoksel/common-words).

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          <li class="site-navigation__item"><a class="site-navigation__link" href="shop.html">Магазин</a></li>
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Контакты</a></li>
+        </ul>
+        <ul class="site-navigation__user-navigation">
```
Неправильное наименование класса по БЭМ. Список не является элементом предыдущего списка.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Контакты</a></li>
+        </ul>
+        <ul class="site-navigation__user-navigation">
+          <li class="site-navigation__login-link"><a href="index.html" aria-label="Вход в учетную запись">Вход</a></li>
```
Здесь то же самое. Неправильный класс для `li`. Про ссылку входа уже писала: почему она ссылается на главную страницу?

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+        <section class="container__gallery gallery">
+          <h2>Фотогалерея</h2>
+          <figure class="gallery__content">
+            <a class="gallery__link" href="#" aria-label="Фотография ссылка на большое изображение"><img src="img/interior.jpg" width="286" height="164" alt="Интерьер барбершопа"></a>
```
Не полностью исправил ошибку с заглавной буквой.
Тут браузер догадывается, что ты хочешь вставить файл Interior.jpg, но лучше на это не полагаться. Тем более, что есть критерий:
Б14. В корне проекта имеются папки css, img, js или аналогичные. Главная страница имеет название index.html. В названиях и расширениях файлов нет заглавных букв и пробелов, использованы только латинские символы.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+        </section>
+      </div>
+
+      <div class="index-columns-2">
```
Вряд ли тут тебе нужен другой класс, так как колонки абсолютно идентичные, как и в прошлом блоке.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+      </nav>
+    </header>
+
+    <main class="container container-centerer">
```
Мне кажется, нет большого смысла делать `main` элементом по БЭМ.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+
+    <footer class="main-footer container-centerer">
+      <div class="main-footer__contacts contacts">
+        <p class="contacts__adress">
```
Опечатка в названии класса.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+      <div class="main-footer__developer developer">
+        <p class="developer__copyright">
+          <b>Разработано:</b>
+          <a class="developer__button button-social" href="https://htmlacademy.ru" aria-label="Ссылка на сайт Академии">HTML-Academy</a>
```
Неправильный класс у ссылки. Она отличается от ссылок на соцсети и по смыслу не относится к ним.
Опечатка в тексте. Должно быть как на макете!

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+    </footer>
 
-    <p>Репозиторий создан для обучения верстке Саньки Петрова.</p>
+    <div class="modal visually-hidden">
```
Тут ты меня, видимо, неправильно понял.
Класс visually-hidden - для элементов, которых в принципе не должно быть видно.
А здесь должны быть класс, который попеременно будет включаться/отключаться.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
 
-    <p>Репозиторий создан для обучения верстке Саньки Петрова.</p>
+    <div class="modal visually-hidden">
+      <div class="wrapper-modal-login">
```
Непонятно, зачем этот `div`, да и класс у него странный и не по БЭМ.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+          <form class="login__form" action="http://echo.htmlacademy.ru" method="post">
+            <input class="login__username" type="text" name="username" placeholder="ЛОГИН" aria-label="Введите ваш логин">
+            <input class="login__password" type="password" name="password" placeholder="ПАРОЛЬ" aria-label="Введите ваш пароль">
+            <input class="login__checkbox" type="checkbox" name="login-radio" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке">
```
А почему radio, если checkbox?

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+            <input class="login__password" type="password" name="password" placeholder="ПАРОЛЬ" aria-label="Введите ваш пароль">
+            <input class="login__checkbox" type="checkbox" name="login-radio" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке">
+            <label class="login__label" for="modal-login-radio">Запомните меня</label>
+            <a class="login__link" href="login.html" aria-label="Ссылка на страницу восстановления пароля">Я забыл пароль!</a>
```
Ты удалил дополнительные страницы, ссылка теперь ведет в никуда.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+            <button class="login__button button" type="submit" aria-label="Кнопка входа в учетную запись">Войти</button>
+          </form>
+        </section>
+        <button class="modal__button button" type="button" aria-label="Кнопка закрытия окна входа на сайт">Закрыть</button>
```
Это надо впихнуть внутрь `section`.

petitechatte replied:
Кстати, класс "button" этой кнопке не подходит, т.к. она отличается от других кнопок на сайте.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+        <button class="modal__button button" type="button" aria-label="Кнопка закрытия окна входа на сайт">Закрыть</button>
+      </div>
 
+      <div class="wrapper-modal-map"></div>
```
То же самое. Непонятно, зачем `div` и почему такой класс.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Барбершоп "Бородинский" - Товар артикул: dexter-ae</title>
```
Артикул - внутренняя информация магазина. Представь, пользователь сохранил страницу в закладки. И еще десять других. А через неделю заходит туда и пытается вспомнить, что это такое.
Кроме того, title выдается в поиске Google. Возможно, даже индексируется. Кто будет искать товар в гугле по артикулу?

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+    <header class="main-header">
+      <nav class="main-header__main-navigation">
+        <a class="main-header__logo-link" href="index.html" aria-label="Ссылка на главную страницу">
+          <img class="main-header__logo logo-header" src="img/logo.png" width="111" height="24" alt="Логотип барбершопа Бородинский">
```
Второй класс по БЭМ выглядит как обозначение блока.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+      </nav>
+    </header>
+
+    <main class="container-item">
```
Чем этот `main` отличается от `main` в каталоге? Зачем ему индивидуальный класс? И точно ли он блок по БЭМ?

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+    </header>
+
+    <main class="container-item">
+      <h1 class="container-item__page-title">Набор для путешествий «Baxter of California»</h1>
```
Здесь то же самое. Заголовки на внутренних страницах выглядят одинаково, их не нужно стилизовать по отдельности.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+        <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="index.html">Главная</a></li>
+        <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="shop.html">Магазин</a></li>
+        <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="#">Средства для ухода</a></li>
+        <li class="breadcrumbs__item--current">Набор для путешествий «Baxter of California»</li>
```
Здесь должно быть два класса: общий и с модификацией.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+        <div class="product-photos__full-container">
+          <img src="img/product-big.jpg" width="486" height="524" alt="Фото всего набора">
+        </div>
+        <div class="product-photos__preview-container">
```
А нужна ли здесь обертка, когда уже есть `ul`?

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+
+    <footer class="main-footer">
+      <div class="main-footer__contacts contacts">
+        <p class="contacts__adress">
```
Опечатка в названии класса.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,116 @@
+      <div class="main-footer__developer developer">
+        <p class="developer__copyright">
+          <b>Разработано:</b>
+          <a class="developer__button button-social" href="https://htmlacademy.ru" aria-label="Ссылка на сайт Академии">HTML-Academy</a>
```
Как и на главной: неправильный класс, опечатка в тексте.

petitechatte commented on index.html:
```diff
@@ -1,12 +1,167 @@
+      <div class="wrapper-modal-map"></div>
+        <section class="modal__map map">
+          <h2 class="visually-hidden">Как до нас добраться</h2>
+          <img src="img/map-popup.png" width="766" height="560" alt="Карта, на которой показано, как к нам проехать">
```
Для карты у тебя сохранены 2 одинаковых изображения. У данного файла название содержит прописную букву, что запрещено базовым критерием.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+        <ul class="main-header__site-navigation site-navigation">
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Информация</a></li>
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Новости</a></li>
+          <li class="site-navigation__item--current"><a class="site-navigation__link" href="#">Прайс-лист</a></li>
```
Ссылка на страницу должна быть неактивной.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+      </nav>
+    </header>
+
+    <main class="container-price">
```
Здесь, возможно, отдельный класс уместен, но его нужно сделать более общим, чтобы подходил под любую страницу, заполняемую контент-менеджером.

petitechatte replied:
Но делать его БЭМ-контейнером все равно не вижу никакого смысла. "Крошки", например, одинаковы на всех страницах.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+      <h1>Прайс-лист</h1>
+      <ul class="container-price__breadcrumbs breadcrumbs">
+        <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="index.html">Главная</a></li>
+        <li class="breadcrumbs__item--current">Прайс-лист</li>
```
Нужно использовать два класса.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+      <ul class="container-price__breadcrumbs breadcrumbs">
+        <li class="breadcrumbs__item"><a class="breadcrumbs__link" href="index.html">Главная</a></li>
+        <li class="breadcrumbs__item--current">Прайс-лист</li>
+      </ul>
```
Вот тут скорее всего нужен общий контейнер, c которым будет работать контент-менеджер через CMS.

petitechatte replied:
Почти все классы ниже нужно будет удалить. Но эта тема еще впереди.

petitechatte replied:
Прощай-прощай БЭМ)))

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+            <li class="partners__item-brand">Suavecito</li>
+            <li class="partners__item-brand">Malin+goetz</li>
+          </ul>
+        <h2 class="resources__header">Цены на услуги <br> наших мастеров:</h2>
```
Я думаю, `br` тут все-таки не нужен.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,124 @@
+        <table class="resources__price price">
+          <tr>
+            <td class="price__service">Стрижка</td>
+            <td class="price__price">1500 &#8381;</td>
```
price__price? Это вообще законно?

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,234 @@
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Информация</a></li>
+          <li class="site-navigation__item"><a class="site-navigation__link" href="#">Новости</a></li>
+          <li class="site-navigation__item"><a class="site-navigation__link" href="price.html">Прайс-лист</a></li>
+          <li class="site-navigation__item--current"><a class="site-navigation__link" href="shop.html">Магазин</a></li>
```
Рекурсивная ссылка

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,234 @@
+          <li class="breadcrumbs__item--current">Средства для ухода</li>
+        </ul>
+
+        <div class="wrapper-products"></div>
```
Странный пустой `div`.

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,234 @@
+                    </a>
+                  </h3>
+                  <p class="products__item-image">
+                    <a href="#" aria-label="Ссылка откроет изображение большего размера">
```
ТЗ 3.13. Карточка товара: изображение товара — это ссылка на страницу товара.

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,234 @@
+                  </p>
+                  <p class="products__item-price">
+                    <b>2900 &#8381;</b>
+                    <a class="products__button" href="#" aria-label="Добавить товар в корзину">Купить</a>
```
Здесь я бы добавила общий класс "button'.

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,234 @@
+                    </a>
+                  </h3>
+                  <p class="products__item-image">
+                    <a href="#" aria-label="Ссылка откроет изображение большего размера">
```
Здесь и далее нужно изменить aria-label: ссылка не открывает изображение, а переносит на страницу с описанием товара.

petitechatte commented on img/form-radio-checked.svg:
```diff
@@ -0,0 +1,10 @@
+    </style>
+  </defs>
+  <path id="Shape_2_copy" data-name="Shape 2 copy" class="cls-1" d="M140,679.976A4.021,4.021,0,1,1,136,684,4.009,4.009,0,0,1,140,679.976Z" transform="translate(-136 -679.969)"/>
+</svg>
```
Это просто круг. Нет смысла добавлять его отдельным файлом.

petitechatte commented on img/form-radio-round.svg:
```diff
@@ -0,0 +1,22 @@
+    <path id="Shape_2_copy_2" data-name="Shape 2 copy 2" class="cls-1" d="M140.009,619A9.992,9.992,0,1,1,130,628.992,10,10,0,0,1,140.009,619Z" style="stroke: inherit; filter: none; fill: inherit"/>
+  </g>
+  <use transform="translate(-129 -618)" xlink:href="#Shape_2_copy_2" style="stroke: #000101; filter: none; fill: none"/>
+</svg>
```
То же самое. Слишком много кода для простой окружности. Можно легко нарисовать в CSS.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	padding: 0;
+	margin: 0;
+	border: 0;
+}
```
Если используешь обнуление, то оно конфликтует с подходом normalize.
По критериям, нужно использовать именно normalize.
Более того, есть еще критерий
Д25. Нет глобальных стилей тегов
Так что отложи своего Андриканича до лучших времен. Еще наиграешься.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	box-sizing: border-box;
+}
+:focus,:active{outline: none;}
+a:focus,a:active{outline: none;}
```
А вот это обнуление заставляет где-то грустить одного Макеева :(

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	-moz-box-sizing: border-box;
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+}
```
Это древнее правило. В мозилле работает без префиксов с 14-го года, на webkit - с 11-го. Такие префиксы уже давно нет смысла использовать.
Как делать box-sizing - смотри критерий Д25.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	height: 100%;
+	width: 100%;
+	font-size: 100%;
+	line-height: 1;
```
Слишком маленький line-height по умолчанию. Строчки будут сливаться.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	width: 100%;
+	font-size: 100%;
+	line-height: 1;
+	font-size: 14px;
```
Это правило перечеркивает верхнее про 100%.
Два правила для одного селектора - нонсенс.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+button::-moz-focus-inner {padding:0;border:0;}
+a, a:visited{text-decoration: none;}
+a:hover{text-decoration: none;}
+ul li{list-style: none;}
```
Вредная фигня, нарушающая критерий Д25.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+ul li{list-style: none;}
+img{vertical-align: top;}
+
+h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: 400;}
```
Д25. Нет глобальных стилей тегов.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+.bigred{
+	font-size: 20px;
+	color: red;
+}
```
Это вообще какой-то личный класс Андриканича, которым ты вряд ли будешь пользоваться.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	text-transform: uppercase;
+
+	background-color: #000000;
+	background-image: url("../img/background-index.png");
```
Фоновое изображение нужно прописать только для главной страницы.

petitechatte replied:
Цвет всех страниц, кроме главной, - белый.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.main-header__main-navigation {
+	font-size: 16px;
+	line-height: 20px;
```
В принципе, line-height здесь прописывать необязательно, так как он наследуется от `body`.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	line-height: 20px;
+	color: #ffffff;
+
+	background-color: #000000;
```
У тебя есть ненужное изображение для хедера. Удали его.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-color: #f8f5f2;
+	background-image: url(../img/background-index-columns.png);
+	background-position: 0 0;
+	background-repeat: repeat;
```
Repeat - поведение фона по умолчанию, можно не указывать. Position для такого "визуального шума" большого смысла тоже не имеет: что ни укажи, ты не заметишь разницы.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-image: url(../img/background-index-columns.png);
+	background-position: 0 0;
+	background-repeat: repeat;
+}
```
Абсолютно одинаковые стили для двух разных селекторов. А раз стили одинаковые, то и класс должен быть один. Зачем плодить сущности?

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+.index-columns h2 {
+	font-size: 30px;
+	line-height: 42px;
+}
```
Не используешь классы там, где это оптимально.
И здесь, и в .features__item h3, и в других местах стили для заголовков одинаковы. Разумно прописать их один раз для одного класса.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.form input:focus {
+	border-color: #663d15;
```
Здесь еще нужно отменить обводку по умолчанию, так как она мешает дизайнерскому фокусу.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-color: #212121;
+	background-image: url(../img/background-footer.png);
+	background-position: 0 0;
+	background-repeat: no-repeat;
```
Почему no-repeat?! Фон должен повторяться. Позицию задвать не обязательно, т.к. это "шум".

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+a {
+	text-decoration: none;
```
Плохая идея, так как в стайлгайде специально показано, что ссылки в тексте подчеркиваются. Значит, убирать подчеркивание по умолчанию нельзя, его нужно отменять для конкретных блоков, например, навигации. Большинство ссылок на макете однотипные, в виде черных кнопок, так что им всем можно дать класс "button" и отменить для него подчеркивание.
P.S. Не слушай Андриканича))

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.main-footer__contacts a {
+	color: #ffffff;
+	text-decoration: underline;
```
Белый цвет нужно задать ВСЕМ ссылкам в хедере и футере, чтобы они автоматически были контрастными, если их добавят в каком-то новом блоке.
Про text-decoration уже писала, он должен быть по умолчанию согласно макету.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.main-footer__contacts a:hover,
+.main-footer__contacts a:focus {
+	text-decoration: none;
```
Это тоже лучше сделать всем ссылкам по-умолчанию.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.button-social {
+	color: #ffffff;
```
Проще и надежнее задать белый цвет ВСЕМ ссылкам в подвале.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+.button-social {
+	color: #ffffff;
+}
+
```
Забыл про инверсию кнопок при наведении!

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-color: #f8f3f0;
+	background-image: url(../img/background-index-columns.png);
+	background-position: 0 0;
+	background-repeat: repeat;
```
Про фон уже писала: 2 последних правила необязательные.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.modal h2 {
+	font-size: 30px;
+	line-height: 42px;
```
И опять эти стили для заголовков, которым не хватает класса...

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+	background-color: #f9f6f3;
+	border: 2px solid #000000;
+}
```
Эти поля очень похожи на поля в форме регистрации. Скорее всего, остальные формы на сайте будут выглядеть также. Стоит подумать, как оптимизировать код, чтобы не писать одно и то же для каждой формы.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.login__form input:focus {
+	border-color: #663d15;
+}
```
Это тоже должно работать для всех полей на сайте.
И нужно убрать обводку по умолчанию.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.login__checkbox:hover,
+.login__checkbox:focus,
```
Чекбоксы и радиокнопки, к сожалению, не поддаются стилизации. Их придется просто прятать.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.login__link {
+	color: #000000;
+	text-decoration: underline;
```
Это должно быть по умолчанию (смотри стайлгайд).

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.login__link:hover,
+.login__link:focus {
+	text-decoration: none;
```
Лучше сделать для всех ссылок.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-color: #f9f6f3;
+	background-image: url(../img/background-index-columns.png);
+	background-position: 0 0;
+	background-repeat: repeat;
```
Здесь стили, одинаковые для всех внутренних страниц. Надо оптимизировать.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.container-shop__page-title {
+	font-size: 30px;
+	line-height: 42px;
```
Типичный заголовок с повторяющимися свойствами - нужен общий класс.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.breadcrumbs a {
+	color: #000000;
```
Все ссылки на сайте по умолчанию должны быть черными, потому что цвет фона по умолчанию - светлый.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.breadcrumbs a:hover,
+.breadcrumbs a:hover {
+	text-decoration: underline;
```
В стайлгайде такое подчеркивание не отрисовано. Можно оставить такой вариант, но мне кажется, более гармонично будет смотреться изменение цвета ссылок, как это происходит с лейблами при наведении.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.filter ul {
+	list-style: none;
+	line-height: 18px;
```
Тут смотри как можно поступить. В стайлгайде есть пример 2 видов списков: сжатый и несжатый. В остальном они выглядят одинаково, с одинаковыми маркерами. Поэтому имеет смысл сделать один общий класс для всех стилизованных списков, и еще дополнительный для изменения межстрочного расстояния.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.filter input:hover,
+.filter input:focus,
```
Чекбоксы и радиокнопки полностью стилизовать не получится, их придется прятать.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	color: #663d15;
+}
+
+.filter input:disabled,
```
То же самое, смысла стилизовать нет.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.filter input:disabled,
+.filter label:disabled {
```
Label не может иметь атрибут disabled, здесь нужно составлять более сложный селектор.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.filter input:disabled,
+.filter label:disabled {
+	color: #000000;
```
Цвет лейблов и так по умолчанию черный. Нужно предусмотреть, чтобы он оставался черным при наведении.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.products__item {
+	background-color: #f8f8f8ff;
```
Цвет карточки чисто белый. На макете видна граница изображения и текстовой части.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.products__item a {
+	color: #000000;
```
Все ссылки должны быть черными по умолчанию.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+	font-family: "PT Sans Narrow", Arial, sans-serif;
+	font-size: 14px;
+	line-height: 20px;
```
В макете в текстовых блоках расстояние между строками указано как 24px!

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.products__item-price b {
+	font-size: 14px;
+	line-height: 20px;
```
На макете 18px, хотя в данном случае это не очень важно.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.products__item-price .products__button {
+	line-height: 20px;
```
Тоже 18px по макету. Во всей карточке интерлиньяж 18.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+.pagination__list a {
+	color: #000000;
+}
+
```
Вот это правило зачем? Оно же переписывается следующим.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	color: #000000;
+	background-color: #ffffff;
+}
+
```
Вместо этого блока можно прописать более точный селектор для активных ссылок.
Не **.pagination__item a:hover**, а **.pagination__item a[href]:hover**.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+	background-color: #f9f6f3;
+	background-position: 0 0;
+	background-repeat: repeat;
```
Те же замечания, что и для каталога: повторяющиеся и избыточные свойства.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-repeat: repeat;
+}
+
+/* !breadcrumbs */
```
Дубликат крошек, которые уже были стилизованы раньше.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	box-shadow: 0 0 10px 2px rgba(0, 0, 0, 0.1);
+}
+
+.product-information__article {
```
Артикул, кстати, по-английски vender code.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.product-information__price b {
+	line-height: 20px;
```
В макете 24px но для цены, которая вряд ли будет в 2 строки, это не очень важно.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.product-information h3 {
+	font-size: 24px;
+	line-height: 30px;
```
Эти размеры уже тоже встречались раньше. Есть смысл для заголовков сделать классы типа header-large, header-medium и т.п.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+.product-information ul {
+	list-style: none;
```
Это типовой сжатый список. У него нет ничего уникального, так что свой класс ему не сильно нужен.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-color: #f9f6f3;
+	background-position: 0 0;
+	background-repeat: repeat;
+}
```
Тут все, как и на других внутренних страницах.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+	background-repeat: repeat;
+}
+
+.container-price b {
```
Это уникальный блок, ему нужен свой класс.

petitechatte replied:
Кстати, а где выравнивание по центру?

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+.container-price b {
+	font-size: 48px;
+	line-height: 48px;
+	font-weight: 700;
```
Это свойство у тебя по умолчанию для всего `body`.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+
+.container-price b {
+	font-size: 48px;
+	line-height: 48px;
```
Здесь имеет смысл пойти против макета и сделать line-height больше, чтобы при переполнении строчки не слипались.

petitechatte commented on css/style.css:
```diff
@@ -0,0 +1,797 @@
+}
+
+/* !Сеточные стили */
+/* index по заданию 6 главы курса */
```
Сетки пока не проверяю, так как я их сейчас не вижу.
Исправь замечания, я приму полуреквест и будем возиться с сетками.

Nomad48JMS committed:
Исправления замечаний. Исправил верстку. И немного по файлам и стилям.

Исправления замечаний. Исправил верстку. И немного по файлам и стилям.

Nomad48JMS committed:
Исправил отсутствие пробелов.

Исправил отсутствие пробелов.

Nomad48JMS committed:
Убрал лишние комментарии.

Nomad48JMS committed:
Правки проекта по 3 пулреквесту.

Правки проекта по 3 пулреквесту.

Nomad48JMS committed:
Вносил поправки и исправлял замечания.

Вносил поправки и исправлял замечания.
